lib/buffer.js

```diff
@@ -874,13 +875,13 @@ Buffer.prototype.write = function(string, offset, length, encoding) {
       length = remaining;
 
     if (string.length > 0 && (length < 0 || offset < 0))
-      throw new RangeError('Attempt to write outside buffer bounds');
+      throw new errors.RangeError('ERR_BUFFER_OUT_OF_BOUND', 'length', true);
```

Comment:
`ERR_BUFFER_OUT_OF_BOUND` -> `ERR_BUFFER_OUT_OF_BOUNDS` With `S` is more common
&nbsp;
![image](https://user-images.githubusercontent.com/96947/27957201-5a78a56e-62eb-11e7-902b-a9556f82135c.png)

![image](https://user-images.githubusercontent.com/96947/27957208-6516a444-62eb-11e7-9659-ed3661d4c039.png)